{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/password.jinja

```diff
@@ -0,0 +1,51 @@
+{% from 'forms/attrs.jinja' import form_attrs %}
+
+{% macro password(name='password', value=None, model="password", attrs={}) %}
+{% do attrs.pop('placeholder', '') %} {# default design doesn't allow for placeholder text, uses labels instead #}
+<span x-data="{input: 'password'}" class="relative">
+    <input :type="input"
+           name="{{ name }}"
+           x-model="{{ model }}"
+           {% if value != None %} value="{{ value }}"{% endif %}
+           {{form_attrs(attrs)}}>
+    <div class="absolute inset-y-0 right-0 pr-9 flex items-center cursor-pointer"
+         @click="input='password'"
+         x-cloak
+         x-show="input==='text'">
+        <svg class="w-6 h-6"
```

Comment:
Looks great, was the heroicons PR Merged ? If so I'd swap those svgs out if possible with them just to reduce the size of this file.

Reply:
no, I will do that though - that would be a good rebase